intranet/templates/eighth/take_attendance.html

```diff
@@ -461,6 +475,9 @@ <h3>Passes</h3>
                 <a class="button print-hide" href="{% url 'eighth_email_students' scheduled_activity.id %}">Email Students</a>
                 {% endif %}
                 {% if user.is_eighth_admin %}
+                    <a class="button print-hide" href="#" onclick="openBbcuTab()">View in BbCU Format</a>
+                    <input type="hidden" id="bbcu_script" value="{{ bbcu_script }}">
```

Comment:
Minor nit
```suggestion
                        <input type="hidden" id="bbcu_script" value="{{ bbcu_script }}">
```

Reply:
Why should this be indented beyond the level that the `<a>` tag above it is at?

Reply:
Disregard this. This was a skim-based rushed review/